client/src/modules/members/model/reducer.js

```diff
@@ -0,0 +1,35 @@
+import { combineReducers } from 'redux';
+
+import { MembersActionTypes } from './actionTypes';
+
+const members = (state = {}, action) => {
+  switch (action.type) {
+    case MembersActionTypes.FETCH_SUCCESS:
+      return { ...state, ...action.payload };
+    case MembersActionTypes.CLEAR_DATA:
+      return {};
+    case MembersActionTypes.ADD_SUCCESS: {
+      const { avatarUrl, displayName, email, newMemberId } = action.payload;
+      return { ...state, [newMemberId]: { avatarUrl, displayName, email } };
```

Comment:
If you change newMemberId to _id on backend, and add to sending data isOwner (see below), you could do:
```
case MembersActionTypes.ADD_SUCCESS: {
  const { _id } = action.payload;
  return { ...state, [_Id]: { ...action.payload } };
```